src/types/production_dependency_config_test.go

```diff
@@ -0,0 +1,46 @@
+package types_test
+
+import (
+	"github.com/Originate/exosphere/src/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ProductionDependencyConfig", func() {
+
+	Describe("validates required production fields", func() {
+		missingConfig := types.ProductionDependencyConfig{
+			Name:    "postgres",
+			Version: "0.0.1",
+			Config: types.ProductionDependencyConfigOptions{
+				Rds: types.RdsConfig{
+					Username:       "test-user",
+					PasswordEnvVar: "TEST_PASSWORD",
+				},
+			},
+		}
+		goodConfig := types.ProductionDependencyConfig{
+			Name:    "postgres",
+			Version: "0.0.1",
+			Config: types.ProductionDependencyConfigOptions{
+				Rds: types.RdsConfig{
+					DbName:         "test",
+					Username:       "test-user",
+					PasswordEnvVar: "TEST_PASSWORD",
+				},
+			},
+		}
```

Comment:
Can we move the creation of these config objects to within the it block that uses them?